smb_exploration/launch/frontier/smb_rss_frontier.launch

```diff
@@ -1,58 +1,26 @@
 <launch>
 
-  <arg name="scan_topic"  default="/scan"                   doc="input topic for the 2D lidar scan"/>
-  <arg name="base_frame"  default="base_link"               doc="frame for the robot base"/>
-  <arg name="odom_frame"  default="tracking_camera_odom"    doc="Odometry frame"/>
-  <arg name="map_frame"   default="map"                     doc="Frame used for publishing the Gmapping Map"/>
-  <arg name="publish_odom_to_map" default="False"           doc="Whether to publish the TF form odom to map or the inverse"/>
+  <arg name="scan_topic"    default="/scan"                   doc="input topic for the 2D lidar scan"/>
+  <arg name="base_frame"    default="base_link"               doc="frame for the robot base"/>
+  <arg name="global_frame"  default="world"                   doc="fixed global frame name in which odometry, map, etc are defined"/>
 
-  <!-- Occupancy Mapping (GMapping) -->
-  <!-- Please refer to the following http://wiki.ros.org/gmapping for details on the parameters-->
-  <node pkg="gmapping" type="slam_gmapping" name="slam_gmapping" output="screen">
-    <param name="base_frame" value="$(arg base_frame)"/>
-    <param name="odom_frame" value="$(arg odom_frame)"/>
-    <param name="map_frame" value="$(arg map_frame)"/>
-    <param name="publish_odom_to_map" value="$(arg publish_odom_to_map)"/>
-    <param name="map_update_interval" value="1.0"/>
-    <param name="maxUrange" value="15.0"/>
-    <param name="maxRange" value="80.0"/>
-    <param name="sigma" value="0.05"/>
-    <param name="kernelSize" value="1"/>
-    <param name="lstep" value="0.05"/>
-    <param name="astep" value="0.05"/>
-    <param name="iterations" value="5"/>
-    <param name="lsigma" value="0.075"/>
-    <param name="ogain" value="3.0"/>
-    <param name="lskip" value="0"/>
-    <param name="minimumScore" value="200"/>
-    <param name="srr" value="0.01"/>
-    <param name="srt" value="0.02"/>
-    <param name="str" value="0.01"/>
-    <param name="stt" value="0.02"/>
-    <param name="linearUpdate" value="0.2"/>
-    <param name="angularUpdate" value="0.3"/>
-    <param name="temporalUpdate" value="-1.0"/>
-    <param name="resampleThreshold" value="0.5"/>
-    <param name="particles" value="80"/>
 
-    <param name="xmin" value="-1.0"/>
-    <param name="ymin" value="-1.0"/>
-    <param name="xmax" value="1.0"/>
-    <param name="ymax" value="1.0"/>
+<!-- Exploration Mapping node -->
+<node name="exploration_map" pkg="smb_exploration" type="exploration_map_node" output="screen">
+  <rosparam file="$(find smb_navigation)/config/move_base_costmaps/costmap_common_params.yaml" command="load" ns="global_costmap" />
+  <rosparam file="$(find smb_exploration)/config/global_costmap_exploration.yaml" command="load"/>
+  <param name="global_costmap/global_frame"     value="$(arg global_frame)"/>
+  <param name="global_costmap/robot_base_frame" value="$(arg base_frame)"/>
```

Comment:
This parameter isn't used / read anywhere in smb_exploration/src/ExplorationMap.cpp if I'm not mistaken.

Reply:
Indeed, deleted